api/index.py

```
@@ -0,0 +1,96 @@
+
+app = FastAPI()
+
+# Initialize Macronizer globally to reuse the instance (and its loaded models/db)
+# This is generally recommended for performance in serverless functions
+# Ensure macronizer.py and its dependencies (db, models) are in the correct relative paths
+try:
+    macronizer_instance = Macronizer()
+except Exception as e:
+    # If Macronizer fails to initialize, the app won't start, which is a critical error.
+    # This helps in debugging issues with finding DB, RFTagger, Morpheus, etc.
+    raise RuntimeError(f"Failed to initialize Macronizer: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)